Description: Do not build examples
 The examples shouldn't be built as part of the package building process, as
 they are meant to be a documentation aid and are installed as such.
 Without this patch, we would be installing built binaries in the package's
 documentation directory. The same applies for benchmarks.
Author: Andreas Hasenack <andreas@canonical.com>
Forwarded: No
Last-Update: 2018-12-05 by kilobyte@angband.pl

--- pmdk-1.5.orig/src/Makefile
+++ pmdk-1.5/src/Makefile
@@ -37,13 +37,13 @@ include $(TOP)/src/common.inc
 
 TARGETS = libpmem libvmem libpmemblk libpmemlog libpmemobj libpmempool\
 		  libvmmalloc tools
-ALL_TARGETS = $(TARGETS) common librpmem examples benchmarks
+ALL_TARGETS = $(TARGETS) common librpmem
 
 SCOPE_DIRS = $(TARGETS) common librpmem rpmem_common
 
 DEBUG_RELEASE_TARGETS = common libpmem libvmem libpmemblk libpmemlog libpmemobj\
 	libpmempool libvmmalloc librpmem
-RELEASE_TARGETS = tools examples benchmarks
+RELEASE_TARGETS = tools
 
 CLEAN_NO_JE_TARGETS = $(ALL_TARGETS) rpmem_common test
 CLEAN_TARGETS = $(CLEAN_NO_JE_TARGETS) jemalloc
--- pmdk-1.5.orig/src/test/Makefile
+++ pmdk-1.5/src/test/Makefile
@@ -319,7 +319,6 @@ LOCAL_TESTS = \
 	$(VMEM_TESTS)\
 	$(VMMALLOC_DUMMY_FUNCS_TESTS)\
 	$(VMMALLOC_TESTS)\
-	$(EXAMPLES_TESTS)\
 	$(LIBPMEMPOOL_TESTS)\
 	$(LIBPMEMPOOL_MOD_TESTS)\
 	$(DAXIO_TESTS)\
